tech/languages/go/go-packages.md

```diff
@@ -29,6 +35,10 @@ The package name idiom is that the import paths of libraries are fully qualified
 
 To install `golang-googlecode-net-devel` package, you use DNF as usual:
 
-```
+```console
 $ sudo dnf install golang-googlecode-net-devel
```

Comment:
```
$ sudo dnf install golang-googlecode-net-devel
Last metadata expiration check: 2:08:10 ago on Fri 17 Jun 2022 12:53:04 PM CEST.
No match for argument: golang-googlecode-net-devel
Error: Unable to find a match: golang-googlecode-net-devel
```

Missed this one when reviewing, according to the fedora page, it was renamed to `golang-x-net-devel`: https://src.fedoraproject.org/rpms/golang-googlecode-net